src/core/comment.ml

```diff
@@ -0,0 +1,153 @@
+(*
+ * Module to manage commentaries
+ * DURAND - MARAIS © 2019
+ *)
+
+
+open Monad
+open Ezjsonm
+
+
+
+(* Module to manage string map *)
+module Map_Str = Map.Make(String)
+
+(* Type commentary *)
+type comment = 
+  | Block of string*string*string
+  | Inline of string
+
+(* Auxiliary constructors *)
+let inline s = Inline s
+let block (s1,s2,s3) = Block (s1,s2,s3)
+
+
+
+(* default commentary list *)
+let default = [
+  ["c" ; "java" ; "go"], Block ("/*", " * ", " */")  ;
+  ["py" ; "sh"], Inline "#" ;
+  ["ml" ; "mli"], Block ("(*", " * ", " *)")  ;
+  [".vim"], Inline "\"";
+]
+
+(* Return an inline value *)
+let get_inline v : comment choice =
+  match v with
+  | `String s -> Ok (inline s)
+  | _ -> Error "[Error] Inline needs a String value"
+
+(* Return a block value *)
+let get_block v =
+  (
+    try Ok (get_list get_string v)
+    with _ -> Error "[Error] Block needs an array of strings"
+  ) >>=
+  (
+    fun l ->
+      try Ok (List.nth l 0 , List.nth l 1, List.nth l 2)
+      with _ -> Error "[Error] Block needs exactly three strings"
+  ) >>= (fun t ->  Ok (block t))
+
+(* Get ext value *)
+let get_ext v =
+  let rec aux l =
```

Comment:
You can use Conf.member and it will take only 3 lines.

Reply:
I can't because it use a (string, value) list